plugins/device-injector/README.md

```diff
@@ -1,16 +1,19 @@
 ## Device Injector Plugin
 
-This sample plugin can inject devices and mounts into containers using
-pod annotations.
+This sample plugin can inject Linux device nodes, CDI devices, and mounts into
+containers using pod annotations.
 
 ### Device Annotations
 
 Devices are annotated using the `devices.nri.io` annotation key prefix.
 The key `devices.nri.io/container.$CONTAINER_NAME` annotates devices to
 be injected into `$CONTAINER_NAME`. The keys `devices.nri.io` and
-`devices.nri.io/pod` annotate devices to be injected into all containers.
+`devices.nri.io/pod` annotate devices to be injected into containers
+without any other, container-specific device annotations. Only one of
```

Comment:
Is the unspecified nature of the ordering not for all three annotations?

Reply:
No. I was way too hasty when addressing your original comment in the docs. It is not unspecified at all which key we pick, since we don't iterate the annotation map, but iterate a slice of key according to our preference/precedence logic. Hence, we always use the more specific annotation (IOW, always only using a container-specific one if present), preferring the explicitly articulated `$key/pod` notation over the plain `$key` one. I updated the docs to reflect this.